### ISSUE TITLE
As a software engineer, I'd like user documentation so that I quickly get information I need.

### ISSUE THREAD

**Issue by aaron-wilson-bfg** [bigfishgames/GameBenchAPI-PyClient#1]
As a software engineer, I'd like user documentation so that I can quickly get the information I need to use the GameBenchAPI client in my Python application.

Done Means:
* README.md has basic integration steps and links to user documentation.
* Sphinx auto-doc for class APIs
* Examples of use
* How to extend the library

**Comment by tristan-fickes-bfg** [comment]
Currently getting feedback for a documentation that includes a summary of the app, steps to use, and an example of how to extend the HTTP library being used.  The link to user documentation will have to wait until we have it fully published on readthedocs.  I will be working on an example app to show it off but found some tasks that may need to be done before then.

**Comment by aaron-wilson-bfg** [comment]
@tristan-fickes-bfg 
https://gamebenchapi-pyclient.readthedocs.io/en/latest/index.html

**Comment by tristan-fickes-bfg** [comment]
Just found that, thanks!

**Comment by tristan-fickes-bfg** [comment]
Since the pull request associated with this was already merged, I created a new issue to track review and updates to the documentation: https://github.com/bigfishgames/GameBenchAPI-PyClient/issues/18
